integration-tests/trait-incrementer/Cargo.toml

```diff
@@ -26,3 +26,7 @@ std = [
     "traits/std",
 ]
 ink-as-dependency = []
+
+[profile.release]
```

Comment:
Please remove those from the examples.

Reply:
why? it overrides default cargo-contract optimization options , to make the contracts smaller

Reply:
Some background: Those options are already passed by `cargo-contract` by default if you don't override the profile (which you just did).

Reply:
nope, cargo-contract passes another default val, that's the point.

Reply:
it sets it to `z`, while `s`seems to work out better (in most contracts)

Reply:
I agree that `lto = true` could be removed. But not the `opt-level = 's'`

Reply:
Yeah I thought we were talking about `z`. lto is enabled, too:
https://github.com/paritytech/cargo-contract/blob/755ea0634d12b4642ab5a2392ea701cdd8da0a4c/crates/build/src/workspace/profile.rs#L31-L39

If we want to change the default we should make the change in cargo-contract and not in the examples. In my eyes it is not a blocker to this PR. Optimizations are chaotic and picking the right level is hard. But if our data suggests that "s" is the better default we should go with that. It is also more conservative which is good.